### ISSUE TITLE
Save related image and masks in Project

### ISSUE THREAD

**Issue by Nodragem** [ankitrohatgi/WebPlotDigitizer#127]
Hello,

I think that it would be nice if the image used for the extraction was saved in the project file so that when we open a project, it also opens the related image.
We could either save a link to the image or embbed the image in the project.

Also, when we have multiple subplots in an image, it would be nice that if we could save one mask by subplot/dataset, and also save the parameters of the extraction (color, delta_x, delta_y) for each subplot/dataset.

Let me know what you think about that.

**Comment by ankitrohatgi** [comment]
Yes, this is something I want to add in the future. A lot of code changes that went in the last release were to facilitate this in the future. I will try to get to this soon.
